apps/api/jwt_auth/handlers.py

```diff
@@ -74,14 +83,24 @@ def jwt_decode_handler(token, get_api_key=APIKey.get_jwt_key):
             leeway=api_settings.JWT_LEEWAY,
             algorithms=[api_settings.JWT_ALGORITHM]
         )
+    except jwt.MissingRequiredClaimError, exc:
+        raise exceptions.AuthenticationFailed(
+            detail=u'Invalid JWT: {}'.format(exc))
```

Comment:
Looks like the exception's message is pretty generic so this is probably cool. Worth logging though, maybe?